index.html

```diff
@@ -132,49 +143,50 @@ <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
             </div>
         </div>
     </section>
-    <div class="aboutme">
+
+    <div class="aboutme" id="About">
         <h3>About me</h3>
-            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-            <a href="" class="get-my-resume">Get My Resume</a>
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="" class="get-my-resume">Get My Resume</a>
     </div>
         <div class="about-technologies">
-        <ul >
-            <li class="aboutme-category">
-                <img src="/images/icon-languages.png" alt="languages">
-                <h3>Languages</h3>
-                <ul class="tech">
-                    <li>Javascipt</li>
-                    <li>Ruby</li>
-                    <li>Html</li>
-                    <li>CSS</li>
-                </ul>
-            </li>
-            <li class="aboutme-category">
-                <img src="/images/icon-frameworks.png" alt="frameworks">
-                <h3>Frameworks</h3>
-                <ul >
-                    <li>Bootstrap</li>
-                    <li>Ruby on rails</li>
-                    <li>RSpec</li>
-                    <li>Capybara</li>
-                    <li>Selenium</li>
-                </ul>
-            </li>
-            <li class="aboutme-category">
-                <img src="/images/icon-skills.png" alt="skills">
-                <h3>Skills</h3>
-                <ul>
-                    <li>Codekit</li>
-                    <li>GitHub</li>
-                    <li>Codepen</li>
-                    <li>Gitlab</li>
-                    <li>Terminal</li>
-                </ul>
-            </li>
-        </ul>
+            <ul >
+                <li class="aboutme-category">
+                    <img src="/images/icon-languages.png" alt="languages">
+                    <h3>Languages</h3>
+                    <ul class="tech">
+                        <li>Javascipt</li>
+                        <li>Ruby</li>
+                        <li>Html</li>
+                        <li>CSS</li>
```

Comment:
- [ ] Kindly follow the design in Figma for the skills section, the badges are not aligned correctly 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/213430177-f58e26b1-ae7b-4fec-8294-6b11a335d45d.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/213430243-8835549c-9bf1-4c79-99b7-f5959d680abd.png)